scripts/irc.js

```diff
@@ -0,0 +1,196 @@
+
+
+/*----------  Function Declaration  ----------*/
+
+const addChatMessage = (author, newMessage) => {
+    let messageToInsert = document.createElement('li');
+    messageToInsert.className = 'irc-modal__chat-message';
+    let messageAuthor = document.createElement('span');
+    messageAuthor.className = 'irc-modal__chat-author';
+    messageAuthor.innerHTML = author + ' ';
+    let messageText = document.createTextNode(newMessage);
+    messageToInsert.appendChild(messageAuthor);
+    messageToInsert.appendChild(messageText);
+
+    ircChatLog.appendChild(messageToInsert);
+}
+
+const greetUser = (event, userMessage) => {
+    const punctuationRegex = /[\u2000-\u206F\u2E00-\u2E7F\\'!"#$%&()*+,\-.\/:;<=>?@\[\]^_`{|}~]/;
+    const spaceRegex = /\s+/g;
+
+    switch (userMessage.toLowerCase().replace(punctuationRegex, '').replace(spaceRegex, ' ')) {
+        case 'hi':
+        case 'hello':
+        case 'good morning':
+        case 'good afternoon':
+        case 'gm':
+        case 'yo':
+        case 'sup':
+        case 'heya':
+        case 'hey':
+        case 'howdy':
```

Comment:
nice use of fall through, an alternative would be an array of possible greetings and you check if the userMessage is contained in there.

Reply:
```js
const possibleGreetings = ['hi', 'hello', 'gm' ] etc 
if(possibleGreetings.some(greeting => greeting === userMessage) {... do stuff }
```